fuzzers/libafl-fuzz/src/feedback.rs

```diff
@@ -0,0 +1,279 @@
+use std::{borrow::Cow, marker::PhantomData, path::PathBuf};
+
+use libafl::{
+    corpus::Testcase,
+    events::EventFirer,
+    executors::ExitKind,
+    feedbacks::{Feedback, FeedbackFactory},
+    inputs::Input,
+    observers::ObserversTuple,
+    state::State,
+};
+use libafl_bolts::{Error, Named};
+use serde::{Deserialize, Serialize};
+
+use crate::Opt;
+
+/// A wrapper feedback used to determine actions for initial seeds.
+/// Handles `AFL_EXIT_ON_SEED_ISSUES`, `AFL_IGNORE_SEED_ISSUES` & default afl-fuzz behavior
+/// then, essentially becomes benign
+#[allow(clippy::module_name_repetitions)]
+#[derive(Debug)]
+pub struct SeedFeedback<A, S>
```

Comment:
I would suggest one file per struct for unrelated stuff, @tokatoka will thank you